compiler-plugin/src/test/kotlin/arrow/meta/quotes/pbt/QuoteTest.kt

```diff
@@ -1,49 +1,79 @@
 package arrow.meta.quotes.pbt
 
+import arrow.meta.plugin.testing.Code
+import arrow.meta.plugin.testing.CompilerTest
+import arrow.meta.plugin.testing.assertThis
+import arrow.meta.quotes.scope.templates.BlockExpressionTest.Companion.blockExpression
+import arrow.meta.quotes.scope.templates.BreakExpressionTest.Companion.breakExpression
+import arrow.meta.quotes.scope.templates.CatchClauseTest.Companion.catchClauseExpression
+import arrow.meta.quotes.scope.templates.ClassBodyTest.Companion.classBodyExpressions
+import arrow.meta.quotes.scope.templates.ContinueExpressionTest.Companion.continueExpressions
+import arrow.meta.quotes.scope.templates.DotQualifiedExpressionTest.Companion.dotQualifiedExpressions
+import arrow.meta.quotes.scope.templates.ForExpressionTest.Companion.forExpressions
+import arrow.meta.quotes.scope.templates.FunctionLiteralTest.Companion.functionalLiteralExpressions
+import arrow.meta.quotes.scope.templates.IfExpressionTest.Companion.ifExpressions
+import arrow.meta.quotes.scope.templates.ImportDirectiveTest.Companion.importDirectiveExpression
+import arrow.meta.quotes.scope.templates.IsExpressionTest.Companion.isExpressions
+import arrow.meta.quotes.scope.templates.LambdaExpressionTest.Companion.lambdaExpressions
+import arrow.meta.quotes.scope.templates.ObjectDeclarationTest.Companion.objectDeclarationExpression
+import arrow.meta.quotes.scope.templates.PackageDirectiveTest.Companion.packageExpressions
+import arrow.meta.quotes.scope.templates.PropertyAccessorTest.Companion.propertyAccessorExpressions
+import arrow.meta.quotes.scope.templates.PropertyTest.Companion.propertyExpressions
+import arrow.meta.quotes.scope.templates.ReturnExpressionTest.Companion.returnExpressions
+import arrow.meta.quotes.scope.templates.ThrowExpressionTest.Companion.throwExpression
+import arrow.meta.quotes.scope.templates.TryExpressionTest.Companion.tryExpression
+import arrow.meta.quotes.scope.templates.TypeAliasTest.Companion.typeAliasExpressions
+import arrow.meta.quotes.scope.templates.TypeReferenceTest.Companion.typeReferenceExpression
+import arrow.meta.quotes.scope.templates.WhenConditionTest.Companion.whenConditionExpression
+import arrow.meta.quotes.scope.templates.WhenEntryTest.Companion.whenEntryExpression
+import arrow.meta.quotes.scope.templates.WhenExpressionTest.Companion.whenExpressions
+import arrow.meta.quotes.scope.templates.WhileExpressionTest.Companion.whileExpression
+import org.junit.Ignore
 import org.junit.Test
 
 class QuoteTest {
 
   @Test
+  @Ignore
```

Comment:
Thanks for this new contribution, @matiaslev!! :tada: Please, I think it would be preferable not to change this file until there is a solution for it. Maybe it's better to remove it. Otherwise, it can lead to confusion.  Thanks!

Reply:
thanks!, I will revert the changes on this file.

Reply:
Thanks to you!! :+1: